docs/README.md

```diff
@@ -50,6 +50,42 @@ Prerequisite: In the API (packages/api) exists a corresponding GraphQL endpoint.
 4) Now the file `index.ts` will be generated automatically `in editor/src/client/api`
 5) Now you can import your desired endpoint in your .tsx file. See for example `subscriptionEditPanel.tsx`
 
+
+### Analytics
+In order to use analytics and send data to WePublish's Matomo, we prepared an analytics.js plugin that's available here:
+
+https://github.com/wepublish/analytics
+
+it integrates with https://github.com/DavidWells/analytics
+
+Usage:
+
+Import and initialise the tracker anywhere in the project
+
+    import  Analytics  from  'analytics'
+    import { matomoPlugin } from '@wepublish/analytics'
+    
+    const  analytics = Analytics({
+	    app: 'Your app name',
+	    plugins: [matomoPlugin()]
+    })
+
+then call the method on page load e.g.
+
+    const { current } = useRoute()
+    
+    useEffect(() => {
+    	analytics.page()
+    }, [current])
+
+The tracker will automatically look in the html structure for an element with a specific id and take data from this element. Example element to send peer tracking data should look like the following: 
```

Comment:
Here I would add where the specific id is retrieved from, I was missing this explanation to understand how it works before asking you.